src/valkey-check-rdb.c

```diff
@@ -280,6 +284,13 @@ int redis_check_rdb(char *rdbfilename, FILE *fp) {
                 goto eoferr;
             }
 
+            if (!strcasecmp(auxkey->ptr, "lua")) {
+                /* In older version before 7.0, we may save lua scripts in a replication RDB,
+                 * although it is not an actually aux field, we will still print it in here since
```

Comment:
What do you mean "it is not an actually aux field"? It is an aux field.

Aux field doens't mean it's not used for anything. I just means that a server ignores it if it doesn't understands it.

Btw, isn't it equally easy to count the lua scripts using `grep --count`?

Reply:
Maybe I just wanted to say that it's not really aux, i.e. it's different from the other aux fields. But i can see the wording is ugly.

I am just bother that we are talking about external grep here a lot, it should be friendly and easy to read on its own without external help. Do you want me to drop it?